modules/taboolaBidAdapter.js

```diff
@@ -233,7 +233,7 @@ function getBid(requestId, currency, bidResponse) {
 
   return {
     requestId,
-    ttl: 360,
+    ttl: 60,
```

Comment:
appears to make your test fail

Reply:
@patmmccann right. thank you! I fixed it